system/spookyhash/context.c

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Is there a reason you're adding the SpookyHash sources directly to our source tree, instead of making it a git submodule ?